content/guides/applications/index.md

```diff
@@ -11,45 +11,29 @@ alias_paths:
 
 # Applications
 
-Box Platform supports two distinctly different application types. The type of
-service you are trying to create determine the type of Box App you will want to configure.
-
 ## Overview
 
-The following is quick overview of the different Box Application types.
+The following is an overview of the different Box Application types you can
+create. 
 
 <ImageFrame border center>
-  ![Application Types](./images/app-types.png)
+  ![Application Types](./images/select-app-type.png)
```

Comment:
For instances like these where you are replacing images you can just replace the existing image rather than creating a new file. It looks like the image is showing up as the old selection page (might be missing the new image checked in): https://staging.developer.box.com/guides/applications/